play-kml-tour/src/main/res/layout/activity_main.xml

```diff
@@ -66,10 +67,10 @@
 
             <com.google.android.material.button.MaterialButton
                 android:id="@+id/playPauseButton"
-                android:layout_width="wrap_content"
+                android:layout_width="0dp"
                 android:layout_height="wrap_content"
-                android:text="@string/play"
                 android:enabled="false"
+                android:text="@string/play"
```

Comment:
The `Play` button text doesn't look like its in center. I think in design, it shows its aligned centrally, but still it looks off weirdly. 

Could we make this text as "Play tour" which actually would clearly mean what we mean by "play" in context here like the "Reset tour" button does while additionally serve the purpose of the center alignment. As the "Play tour" text looks more centrally aligned.